charts/portworx-essentials/v1.0.0/README.md

```diff
@@ -0,0 +1,36 @@
+# Portworx
```

Comment:
As a completely new user of Portworx, I had some trouble figuring out the bare minimum needed to get the chart installed. It would be nice to include here or in app-readme (if brief), some instructions to at least get a basic setup installed. Some of the things I had to figure out were:
1. This chart needs to be installed in the system part of your cluster since the chart is locked to the "kube-system" namespace and default has no access to that namespace.
2. My understanding is you can have a variety of different configurations for Portworx, but the bare minimum that will get you a full green everything active install is having a cluster with 3 nodes, and at least 1 volume in one of them. It would be helpful to document this somewhere since as a new user, even though Portworx can work with less, I would still wonder if is working correctly and spend time figuring out why Stork is not installing everything correctly.
3. The default username/password for the UI should be here.

Reply:
Added more details to the README.md